pkgs/tools/audio/google-music-scripts/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, python3, substituteAll, locale }:
+
+let
+  click6 = python3.pkgs.click.overridePythonAttrs (oldAttrs: rec {
+    version = "6.7";
+
+    src = python3.pkgs.fetchPypi {
+      pname = "click";
+      inherit version;
+      sha256 = "02qkfpykbq35id8glfgwc38yc430427yd05z1wc5cnld8zgicmgi";
+    };
+
+    patches = lib.optional (lib.versionAtLeast version "6.7") (substituteAll {
```

Comment:
This is in the click package already for the exact purpose of overriding it to `6.7`. Any reason why it's here?

Reply:
You mean the version check? Sorry that was copy-pasted.